src/pages/ProgramCollectionAdminPage.tsx

```diff
@@ -373,7 +373,12 @@ const useProgramPreviewCollection = (
               condition: {
                 ...condition,
                 ...(Object.keys(orderBy ? orderBy[0] : {})[0] === 'position'
-                  ? { position: { _gt: data?.program.slice(-1)[0]?.position } }
+                  ? {
+                      _or: [
+                        { position: { _gt: data?.program.slice(-1)[0]?.position } },
+                        { id: { _gt: data?.program.slice(-1)[0]?.id } },
+                      ],
+                    }
```

Comment:
i thought the condition should be `_and`?

Reply:
I found some logical conflict
add another condition to resolve it